Assets/Scripts/Augment/GunController.cs

```diff
@@ -60,6 +59,13 @@ private void FireGun()
         }
         stats.Ammo--;
 
+        // Aim at target but lerp in original direction if target is close
```

Comment:
This should probably be in onFire instead to avoid the raycast getting stuck while holding the fire button

Reply:
Good catch, fixed!